src/shork.gleam

```diff
@@ -209,6 +224,7 @@ pub fn run_query(
   a: Connection,
   b: String,
   c: List(Value),
+  d: option.Option(Int),
 ) -> Result(#(List(String), List(dynamic.Dynamic)), QueryError)
```

Comment:
I think run_query should not be public. Especially when stuff like the timeout that should be encapsulated in the Query object are used as parameters.